lib/server.js

```diff
@@ -34,6 +34,7 @@ function handleResponse(opts, req, resp, response) {
         rh['access-control-allow-origin'] = '*';
         rh['access-control-allow-methods'] = 'GET';
```

Comment:
Actually, it's not needed. The [spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Preflighted_requests) says:

> Cross-site requests are preflighted like this since they may have implications to user data.  In particular, a request is preflighted if:
> - It uses methods other than GET, HEAD or POST.  Also, if POST is used to send request data with a Content-Type other than application/x-www-form-urlencoded, multipart/form-data, or text/plain, e.g. if the POST request sends an XML payload to the server using application/xml or text/xml, then the request is preflighted.
> - It sets custom headers in the request (e.g. the request uses a header such as X-PINGOTHER)